docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -277,6 +279,74 @@ fixture `MyFixture`
     .page `../my-project/index.html`;
 ```
 
+## Specifying Testing Metadata
+
+TestCafe allows you to specify additional information for tests in the form of *key-value metadata* and use it in reports.
+
+To specify metadata, use the `meta` method. You can call this method both for a fixture and a test.
+
+* You can specify one metadata entry for a fixture or a test.
+
+    ```js
+    fixture.meta('key1', 'value1')
+    ```
+
+    ```js
+    test.meta('key2', 'value2')
+    ```
+
+    Parameter | Type   | Description
+    --------- | ------ | -----------------
+    `name`    | String | The name of the metadata entry
+    `value`   | String | The value of the metadata entry
+
+* You can specify a set of metadata entries for a fixture or a test.
```

Comment:
I don't like these two points:

```
* You can specify one metadata entry for a fixture or a test.
* You can specify a set of metadata entries for a fixture or a test.
```

After reading the first one, I begin thinking that I can specify **only** one entry. I personally got surprised. And then I see the second point that contradicts the first one.

Let's either add `either`...

```
* You can either specify one metadata entry for a fixture or a test...
* ...Or a set of metadata entries.
```

**(but that's not quite good)**

... or add a phrase before this list.

```
To specify metadata, use the `meta` method. You can call this method both for a fixture and a test.

The 'meta' method allows you to specify one or several meta entries.

* Specifying one metadata entry for a fixture or a test.

blablabla

* Specifying a set of metadata entries for a fixture or a test.

blablabla
```

or even 

```
### Specifying one metadata entry for a fixture or a test

blablabla

### Specifying a set of metadata entries for a fixture or a test

blablabla
```